core/types/rlp_backwards_compat.libevm_test.go

```diff
@@ -29,6 +29,34 @@ import (
 )
 
 func TestHeaderRLPBackwardsCompatibility(t *testing.T) {
+	tests := []struct {
+		name     string
+		register func()
+	}{
+		{
+			name:     "no registered extras",
+			register: func() {},
+		},
+		{
+			name: "no-op header hooks",
+			register: func() {
+				RegisterExtras[NOOPHeaderHooks, *NOOPHeaderHooks, struct{}]()
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			TestOnlyClearRegisteredExtras()
+			defer TestOnlyClearRegisteredExtras()
+			tt.register()
+			testHeaderRLPBackwardsCompatibility(t)
+		})
+	}
+}
+
+//nolint:thelper
+func testHeaderRLPBackwardsCompatibility(t *testing.T) {
```

Comment:
supernit (supernit = feel free to discard & ignore this comment) why not having it inlined in `TestHeaderRLPBackwardsCompatibility` since it's used only once?

Reply:
Because of the sensitivity of changes to the test data I wanted to make it 100% clear that nothing was changing. I originally did it as you mention but the GitHub rendering of the diff was horrendous.